tests/console/consoletest_finish.pm

```diff
@@ -74,7 +74,14 @@ sub run() {
         }
         wait_idle;
         mouse_hide(1);
-        assert_screen 'generic-desktop';
+        # in case of awesome test we use xdm and need to log in at the display
+        # manager
```

Comment:
this is wrong. The login into xdm is like 12 lines earlier - at this point we're supposed to be in the desktop - or the mouse_hide is pointless